javascript/ql/lib/semmle/javascript/Concepts.qll

```diff
@@ -110,3 +110,40 @@ abstract class PersistentWriteAccess extends DataFlow::Node {
    */
   abstract DataFlow::Node getValue();
 }
+
+/**
+ * Provides models for cryptographic things.
+ */
+module Cryptography {
+  private import semmle.javascript.internal.ConceptsShared::Cryptography as SC
+
+  /**
+   * A data-flow node that is an application of a cryptographic algorithm. For example,
+   * encryption, decryption, signature-validation.
+   *
+   * Extend this class to model new APIs. If you want to refine existing API models,
+   * extend `CryptographicOperation` instead.
```

Comment:
> Extend this class to model new APIs. 

That doesn't sound right, it sounds like you've copied from `CryptographicOperation::Range` instead of `CryptographicOperation`.

Reply:
Thanks